static/js/around-map.js

```diff
@@ -0,0 +1,46 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoibmVvbWFkIiwiYSI6ImNqMHRrZ3ZwdzAwNDgzMm1kcHRhMDdsZGIifQ.bOSlLkmc1LBv0xAbcZXpog';
+const map = new mapboxgl.Map({
+  container: 'map',
+  style: 'mapbox://styles/mapbox/streets-v9',
+  center: [-10, 45],
+  zoom: 2
+})
+
+// users interesting points
+const giveData = new Worker("/static/js/webworker-around.js");
+
+giveData.onmessage = (informations) => {
```

Comment:
parenthesis are not required here. For consistency – because they are not used elsewhere in the code when it's a unique parameter – they need to be removed.